specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,661 @@
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "extendedLocation": {
+          "$ref": "./CommonDefinitions.json#/definitions/ExtendedLocation"
+        },
+        "identity": {
+          "description": "managed identities for the Container App to interact with other Azure services without maintaining any secrets or credentials in code.",
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
+        "properties": {
+          "args": {
+            "items": {
+              "type": "string"
+            },
+            "type": "array"
+          },
+          "command": {
+            "items": {
+              "type": "string"
+            },
+            "type": "array"
+          },
+          "provisioningState": {
+            "description": "Provisioning state of the Container Apps Job.",
+            "enum": [
+              "InProgress",
+              "Succeeded",
+              "Failed",
+              "Canceled",
+              "Deleting"
+            ],
+            "type": "string",
+            "readOnly": true,
+            "x-ms-enum": {
+              "name": "ContainerAppsJobProvisioningState",
+              "modelAsString": true
+            }
+          },
+          "managedEnvironmentId": {
+            "description": "Deprecated. Resource ID of the Container App's environment.",
+            "type": "string",
+            "x-ms-mutability": [
+              "create",
+              "read"
+            ]
+          },
+          "environmentId": {
+            "description": "Resource ID of environment.",
+            "type": "string",
+            "x-ms-mutability": [
+              "create",
+              "read"
+            ]
+          },
+          "configuration": {
+            "$ref": "#/definitions/Configuration",
+            "description": "Non versioned Container Apps Job configuration properties."
+          },
+          "template": {
+            "$ref": "./CommonDefinitions.json#/definitions/ContainerAppsJobTemplate",
+            "description": "Container App versioned application definition."
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ContainerAppsJobsCollection": {
+      "description": "Container Apps Jobs collection ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerAppsJob"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
```

Comment:
Did we decide not so support `Dapr`? https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2022-11-01-preview/specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerApps.json#L641

Reply:
Yes it is V2. We should remove Dapr below from V1 OR allow ingress.

Reply:
Dapr is not necessarily tied to ingress. We can still support Dapr even with no ingress - let's meet with the Dapr team to clarify.

Reply:
@trajkobal have kept dapr component for now in the spec as per the discussion.